src/coreclr/nativeaot/Runtime/gcenv.ee.cpp

```diff
@@ -533,54 +533,84 @@ struct ThreadStubArguments
 {
     void (*m_pRealStartRoutine)(void*);
     void* m_pRealContext;
-    bool m_isSuspendable;
     CLREventStatic m_ThreadStartedEvent;
 };
 
+static bool CreateUnsuspendableThread(void (*threadStart)(void*), void* arg, const char* name)
```

Comment:
```suggestion
static bool CreateNonSuspendableThread(void (*threadStart)(void*), void* arg, const char* name)
```

Reply:
I think it is better English, and it matches the equivalent method name in regular CoreCLR.